Client/Frontend/DevicePickerTableViewCell.swift

```diff
@@ -41,20 +42,21 @@ class DevicePickerTableViewCell: UITableViewCell, ReusableCell, ThemeApplicable
     override func layoutSubviews() {
         super.layoutSubviews()
 
-        nameLabel.snp.makeConstraints { (make) -> Void in
-            make.left.equalTo(UX.deviceRowTextPaddingLeft)
-            make.centerY.equalTo(self.snp.centerY)
-            make.right.equalTo(self.snp.right).offset(-UX.deviceRowTextPaddingRight)
-        }
+        NSLayoutConstraint.activate([
+            nameLabel.leadingAnchor.constraint(equalTo: leadingAnchor, constant: UX.deviceRowTextPaddingLeft),
+            nameLabel.centerYAnchor.constraint(equalTo: centerYAnchor),
+            nameLabel.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -UX.deviceRowTextPaddingRight)
+        ])
```

Comment:
I also think there is room for improvement here. We don't need to activate the constraints with every layout change; instead, we could create a `setupLayout()` method, add the constraints there, and call it in the `init`. This change is arbitrary and not related to this PR, so feel free to consider it only if you want. Thank you for all the good work you put in and your constant PRs.